Code/Framework/AzCore/AzCore/RTTI/AzStdOnDemandReflection.inl

```diff
@@ -106,6 +106,8 @@ namespace AZ
             if (behaviorContext)
             {
                 behaviorContext->Class<ContainerType>()
+                    ->Attribute(AZ::Script::Attributes::Scope, AZ::Script::Attributes::ScopeFlags::Common)
+                    ->Attribute(AZ::Script::Attributes::Module, "std")
```

Comment:
Does this mean existing scripts will need the `std::` prepended?

Reply:
This provides access to the type in python via `import azlmbr.std.*`